spec/std/isa/inst_subtype/I/I-x-x.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2025 Usman Akinyemi
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/inst_subtype_schema.json
+
+$schema: inst_subtype_schema.json#
+kind: instruction_subtype
+name: I-x-x
+
+data:
+  type: { "$ref": "inst_type/I.yaml#" }
+  subtype: { "$ref": "inst_subtype/I/I-x-x.yaml#" }
+  opcodes:
+    $inherits: inst_type/I.yaml#/opcodes
+  variables:
+    func11:
+      $inherits: inst_var/i-imm12.yaml#/data
```

Comment:
Bike-shedding, perhaps...
Since the `i` in `i-imm12` references the instruction type, `I`, perhaps it should be capitalized to match: `I-imm12`.
Further, if we don't expect other lengths for an I immediate, perhaps it should be `I-imm`? (Or, even if we do expect other lengths, this 12-bit length will surely dominate and the others would be exceptions to the rule and can be named as such if and when the need arises.)

Reply:
The only obvious exception is when that 12-bit "imm" field is broken up into 2 fields for shift instructions, and those will have to be new variables (and opcodes!) anyway, with names like "shamt" instead of "imm".

Reply:
`I-imm`, please.